### DISCUSSION TITLE
How to detect when scroll did end ?

### DISCUSSION THREAD

**Discussion by jserrafindster** [xmartlabs/Eureka#2224] [Q&A]
I programmatically make the form scroll by using the method `UITableView.scrollToRect()`, but I can't find a way to detect when the scroll has ended, this method has no completion. 
So I've tried to use the `UITableViewDelegate` with form.tableview but this causes the form to lose its behaviour/properties.

I also tried to search the `FormDelegate` but it seems it has no methods to observe the scroll.

What is the correct way to detect when the scroll has ended?

**Comment by mats-claassen** [comment]
Your FormViewController is already the delegate of form.tableView. So you should be able to implement any of the methods in UITableViewDelegate protocol in that class. Keep in mind though to call super if you override a method that is implemented in FormViewController
